src/components/v5/common/Table/Table.tsx

```diff
@@ -0,0 +1,145 @@
+import React, { useState } from 'react';
+import {
+  flexRender,
+  getCoreRowModel,
+  useReactTable,
+  createColumnHelper,
+} from '@tanstack/react-table';
+import { useFieldArray, useFormContext } from 'react-hook-form';
+import clsx from 'clsx';
+import { v4 as uuidv4 } from 'uuid';
+import { TableProps } from './types';
+import Button from '~v5/shared/Button';
+import { useMobile } from '~hooks';
+import BurgerMenu from '~v5/common/Table/partials/BurgerMenu';
+
+const displayName = 'v5.common.Table';
+
+const Table = <T,>({
```

Comment:
I am a bit confused. This component seems to be doing too many things at once. This should be only a base where the styles are added but the columns should be passed from outside. If you have a table that has burger menu reapeating across the project you should just create another component that is using this one inside of it.